src/sentry/tsdb/base.py

```diff
@@ -413,16 +415,17 @@ def delete(
     def get_range(
         self,
         model: TSDBModel,
-        keys: list[int],
+        keys: Sequence[TSDBKey],
         start: datetime,
         end: datetime,
         rollup: int | None = None,
         environment_ids: list[int] | None = None,
+        conditions=None,
```

Comment:
idk what this `conditions=` is -- but it's part of the signature for the `snuba` subclass and these need to match otherwise they're not substitutable 🤷